test-utils/throttleMock.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function() {
+  var queue = []
+  return {
+    throttle: function(fn) {
```

Comment:
Is there a particular reason why the below closure is necessary (as opposed to a direct call)? It just seems odd to me.